update-css/README.md

```diff
@@ -0,0 +1 @@
+# Recipe: Cache and update CSS
```

Comment:
Believe it or not, most of the problem is here. The recipe is illustrating at least, two techniques (messaging and using cache as database) non related with #9 . Defining the use case and the solution overview help you to keep the recipe focused in what you want to solve.

In #9 we are talking about the problem of caching **automatically** large files (fonts) and almost long term immutable (images and CSS) assets. This should lead to better loading times, for instance (and we could demonstrate comparing first load with the subsequent ones) and a way to manually wipe out the cache to repeat the experiment.

The solution would be to perform a generic fetch strategy that detects CSS, image and font fetching, and try to match the cache, then fetch the network to obtain a fresh response for the next serving.

Although _recipe_ could point to a mixture of techniques to solve a problem, I think recipes should not show more than one set of features (CSS case is to inspect request properties to take decisions for fetching or not) at a time.

Messaging and the cache-as-db trick **are great** (I use the latter in offliner) but are not mainly related with caching large / semi static files. More even when, for messaging, the SW only acts like a pass through channel for messages and as a message handler; furthermore, you will realize that it is not as simple to get a working solution for Nightly and Chrome.

So, summarizing, my humble opinion is: write the use case, write the overview for the solution and design an experiment to make the user realize about the value of your proposal.

Then try to stick with it and introduce as fewer noise as possible.

Let me know what you think.